test/gtest/uct/uct_test.cc

```diff
@@ -142,33 +142,56 @@ std::vector<uct_test_base::md_resource> uct_test_base::enum_md_resources() {
 
 uct_test::uct_test() {
     ucs_status_t status;
-    uct_md_attr_t pd_attr;
-    uct_md_h pd;
+    uct_md_attr_t md_attr;
```

Comment:
should have been another PR "TEST/UCT: Rename pd to md"

Reply:
why..? 
it's just one param rename in one test function...

do you want this split?